src/d/d_item_data.cpp

```diff
@@ -3,26 +3,10152 @@
+    "VmapG",
+    "VboxA",
+    "Vtin1",
+    "Vtin2",
+    "Vtin3",
+    "Vtin4",
+    "Vtin5",
+    "Vbeso",
+};
+
+/* 803840E0-803842B0       .data item_texture_tbl__10dItem_data */
+char* dItem_data::item_texture_tbl[0x74] = {
+    "get_rupy.bti",
+    "get_key.bti",
+    "heart_up_01.bti",
+    "heart_up_02.bti",
+    "telescope.bti",
+    "whistle.bti",
+    "baton.bti",
+    "camera.bti",
+    "coverofbeast.bti",
+    "boomerang.bti",
+    "rope.bti",
+    "sail_00.bti",
+    "fan.bti",
+    "hammer_01.bti",
+    "boots_00.bti",
+    "boots_01.bti",
+    "coverofbait.bti",
+    "bow_01.bti",
+    "bomb_00.bti",
+    "sail_01.bti",
+    "bottle_00.bti",
+    "bottle_01.bti",
+    "bottle_02.bti",
+    "bottle_03.bti",
+    "delivery.bti",
+    "hookshot.bti",
+    "shield_02.bti",
+    "sail_02.bti",
+    "beast_01.bti",
+    "beast_02.bti",
+    "beast_03.bti",
+    "beast_04.bti",
+    "beast_05.bti",
+    "beast_06.bti",
+    "beast_07.bti",
+    "beast_08.bti",
+    "bait_01.bti",
+    "bait_02.bti",
+    "bait_03.bti",
+    "bait_04.bti",
+    "bait_05.bti",
+    "bait_06.bti",
+    "bait_07.bti",
+    "bait_08.bti",
+    "baton_stone.bti",
+    "god_symbol_00.bti",
+    "god_symbol_01.bti",
+    "god_symbol_02.bti",
+    "sword_00.bti",
+    "sword_01.bti",
+    "shield_00.bti",
+    "shield_01.bti",
+    "gloves_00.bti",
+    "gloves_00.bti",
+    "amulet_00.bti",
+    "amulet_01.bti",
+    "delivery_01.bti",
+    "delivery_02.bti",
+    "delivery_03.bti",
+    "delivery_04.bti",
+    "delivery_05.bti",
+    "delivery_06.bti",
+    "delivery_07.bti",
+    "delivery_08.bti",
+    "delivery_09.bti",
+    "delivery_10.bti",
+    "delivery_11.bti",
+    "delivery_12.bti",
+    "delivery_13.bti",
+    "delivery_14.bti",
+    "delivery_15.bti",
+    "delivery_16.bti",
+    "delivery_17.bti",
+    "delivery_18.bti",
+    "dungeon_map.bti",
+    "compass.bti",
+    "boss_key.bti",
+    "bottle_04.bti",
+    "bottle_05.bti",
+    "bottle_06.bti",
+    "bottle_07.bti",
+    "bottle_08.bti",
+    "baton_stone.bti",
+    "triforce_00.bti",
+    "triforce_01.bti",
+    "triforce_02.bti",
+    "triforce_03.bti",
+    "triforce_04.bti",
+    "triforce_05.bti",
+    "triforce_06.bti",
+    "triforce_07.bti",
+    "arrow_00.bti",
+    "arrow_01.bti",
+    "arrow_02.bti",
+    "arrow_03.bti",
+    "big_purse.bti",
+    "max_purse.bti",
+    "sword_02.bti",
+    "sword_03.bti",
+    "clothes.bti",
+    "bombpouch_1.bti",
+    "bombpouch_2.bti",
+    "arrowcase_1.bti",
+    "arrowcase_2.bti",
+    "cmap_tri2.bti",
+    "cmap_treasure2.bti",
+    "cmap_treasure2.bti",
+    "cmap_phantomship2.bti",
+    "cmap_tingle2.bti",
+    "arrow_power_01.bti",
+    "arrow_power_02.bti",
+    "camera_2.bti",
+    "tingle_figure.bti",
+    "bottle_09.bti",
+    "cmap_hint2.bti",
+    "delivery_19.bti",
+};
+
+/* 803842B0-803866B0       .data item_resource__10dItem_data */
+dItem_data_item_resource dItem_data::item_resource[0x100] = {
+    {
+        // 0x00
```

Comment:
iirc each entry in here is related to the corresponding item in the item table, so it might be nice to comment which the item name here, but its not a super important thing